### DISCUSSION TITLE
Is there a way to get ant-design-mobile work with nextjs?

### DISCUSSION THREAD

**Discussion by koeit** [ant-design/ant-design-mobile#5250] [Q&A] [answered]
I installed ant-design-mobile on a nextjs project and getting an Error when I try to use components like e. g. NavBar:

`./node_modules/antd-mobile/es/components/action-sheet/action-sheet.css
Global CSS cannot be imported from within node_modules.
Read more: https://nextjs.org/docs/messages/css-npm
Location: node_modules\antd-mobile\es\components\action-sheet\index.js`

is there a way to fix this?

**Comment by awmleer** [comment, marked as answer]
Have you tried [this](https://mobile.ant.design/guide/ssr#nextjs)?

**Reply by koeit** [reply]
That´s it, thank you very much!
